frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellResource.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { LoadingBox } from '@console/internal/components/utils';
+import CloudShellResource from '../CloudshellResource';
+
+let CloudShellWrapper: ShallowWrapper;
+type CloudShellResourceProps = React.ComponentProps<typeof CloudShellResource>;
+const cloudShellResourceProps: CloudShellResourceProps = {
```

Comment:
These should be declared in the `describe` block and defined in the `beforeAll` or `beforeEach` block.

For your unit tests, you can define these in the `beforeEach` block and create two `it` test case with there own shallow render.